client/src/views/Home/MovieItems/styles.scss

```diff
@@ -0,0 +1,45 @@
+@import "../../../styles/master";
+.list-container {
+  div {
+    border-radius: 0.25rem;
+    position: relative;
+    img {
+      border-radius: 0.25rem;
+      height: 100%;
+      width: 100%;
+    }
+    .movie-layer {
+      position: absolute;
+      left: 0;
+      top: 0;
+      height: 100%;
+      width: calc(100% - 16px);
+      background-color: $shade-disabled;
+      opacity: 0;
+      color: $shade-default;
+
+      &:hover {
+        opacity: 0.9;
+        color: $shade-default;
+      }
+
+      img {
+        width: 1.25rem;
+        height: auto;
+      }
+
+      div {
+        p {
```

Comment:
The deeply nested class name is not a good practice in CSS, it's better to use a lot of specific class names than creating so many levels on selectors.